examples/sfml/main.cpp

```diff
@@ -0,0 +1,16 @@
+#include <SFML/Graphics.hpp>
+#include <SFML/Window.hpp>
```

Comment:
`<SFML/Graphics.hpp>` includes `<SFML/Window.hpp>` so that 2nd header can be removed.